modules/adspendBidAdapter.js

```diff
@@ -0,0 +1,120 @@
+import * as utils from 'src/utils';
+import { config } from 'src/config';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER, NATIVE, VIDEO } from 'src/mediaTypes';
+
+const BIDDER_CODE = 'adspend';
+const URL = '//rtb.com.ru/headerbidding-bid';
+const COOKIE_NAME = 'hb-adspend-id';
+const UUID_LEN = 36;
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['adspend'],
+  supportedMediaTypes: [BANNER, VIDEO, NATIVE],
```

Comment:
Are you planning to support all these mediaTypes?  The rest of the code seems to support only `BANNER`.  If that's the case - I would suggest to only list the types that are currently supported in the adapter and update it later when the additional support is added.

Reply:
Yes, as I said before, we're going to support all these mediaTypes, but currently we don't. I'll remove unsupported mediaTypes for now.